src/database.js

```diff
@@ -128,6 +128,32 @@ function setUpModels(db) {
             }
         }
     });
+    models.Pod.homeStats = function (callback) {
+        db.driver.execQuery(
+          "SELECT \
+            (SELECT COUNT(*) FROM pods) AS total_nodes,\
```

Comment:
All of these queries should skip pods which have had too many failures, otherwise we show all the dead nodes too. So `pods.failures < 3` to all the selects. You can see the problem in the current dev site, showing over 700 active nodes - which would be nice but it's false ;)

We should probably add indexes to the tables, starting to have lots of data and more selects to render the page, I'll make a separate issue re that.